### PR TITLE
feat: added config to poyfill node js modules

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Added config to support nodejs modules that are not supported in webpack 5

for context see the thread 
https://discord.com/channels/664594912460668960/929954765021011978/1018852026932801586

### CODE REVIEW

File: config-overrides.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        http: require.resolve("stream-http"),
+        https: require.resolve("https-browserify"),
+        util: require.resolve("util"),
+        url: require.resolve("url/"),
+    });
+    config.resolve.fallback = fallback;
+    config.plugins = (config.plugins || []).concat([
+        new webpack.ProvidePlugin({
+            process: "process/browser",
+            Buffer: [require.resolve("buffer/"), "Buffer"]
```
[the remark this answers is not in the document]
Comment (aurelticot) [reply config-overrides.js:14]: Also, I've seen and used other configs where it was just `Buffer: ["buffer", "Buffer"],`.
Is it ok with the `require.resolve("...")` part?

**Review by aurelticot — CHANGES_REQUESTED** [review]
Just a few questions.
Otherwise ok

File: config-overrides.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmcWebCode40) [reply config-overrides.js:14]: yes its actually important and its specified here https://www.npmjs.com/package/buffer#usage

> Is it ok with the require.resolve("...") part?
Yes it is works as expected

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "react-app-rewired": "^2.2.1",
     "react-copy-to-clipboard": "^5.1.0",
     "react-dom": "^18.2.0",
-    "react-loader-spinner": "^5.1.0",
+    "react-loader-spinner": "^5.3.4",
```
Comment (aurelticot) [inline package.json:17]: As we removed the import in `src/index.js`, do we actually use this dependency somewhere?
If not, we should remove it completely

**Review by aurelticot — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (cmcWebCode40) [reply package.json:17]: @aurelticot,

The dependency was used in this view component here 

https://github.com/verida/react-stater-kit/blob/7ccad096974a6f31650baea21216561d52d356f7/src/pages/Connect.jsx#L2

Comment (aurelticot) [reply package.json:17]: Was or still used?
If it is still, ok, we keep it.
If it was but no longer is, then we remove it.

Comment (cmcWebCode40) [reply package.json:17]: Yes is still used

Comment (aurelticot) [reply package.json:17]: ok

**Review by aurelticot — APPROVED** [review]

**Pull request merged**